apps/andi/test/unit/andi/migration/migrations_test.exs

```diff
@@ -10,9 +10,10 @@ defmodule Andi.Migration.MigrationsTest do
   @modified_date_event "migration:modified_date:start"
 
   alias Andi.Migration.Migrations
+  alias Andi.Services.DatasetRetrieval
 
   test "send the modified date migration event if it has not succeeded yet" do
-    allow(Brook.get!(@instance, :migration, "modified_date_migration_completed"), return: nil)
+    allow(DatasetRetrieval.get!("modified_date_migration_completed", :migration), return: nil)
```

Comment:
This doesn't seem like it should have changed.  Seems like we should only be wrapping calls to brook that look at the `:dataset` or `:ingested_time` collections.